data/pg/works/ContractOfficerIncharge.json

```diff
@@ -0,0 +1,12 @@
+{
+  "tenantId": "pg",
+  "moduleName": "works",
+  "OICRoles": [
```

Comment:
is wrong

Reply:
file name and master name should be same

Reply:
if file name is ContractOfficerIncharge.json
then your array also should have same name
ie "ContractOfficerIncharge" : []